Backpack/Icon/Classes/BPKIcon.m

```diff
@@ -34,6 +34,10 @@ @implementation BPKIcon
 + (UIImage *)templateIconNamed:(NSString *)name size:(BPKIconSize)size {
     UIImage *image = [self iconNamed:name color:UIColor.blackColor size:size];
 
+    if ([BPKAutoMirrorIconNames.items containsObject:name]) {
```

Comment:
should this be inside the `iconNamed` method so that all icons have it and not just templates?
not sure if this would even apply tho

Reply:
Yes, that's a good catch!